pkg/querier/querier.go

```diff
@@ -28,6 +28,7 @@ type Config struct {
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	flag.IntVar(&cfg.MaxConcurrent, "querier.max-concurrent", 20, "The maximum number of concurrent queries.")
 	flag.DurationVar(&cfg.Timeout, "querier.timeout", 2*time.Minute, "The timeout for a query.")
+	flag.DurationVar(&promql.LookbackDelta, "querier.lookback-delta", promql.LookbackDelta, "The delta difference allowed for retrieving metrics during expression evaluations.")
```

Comment:
This description doesn't mean much to me, and I kinda know what this does.

I thought the one in the comment was ok - "LookbackDelta determines the time since the last sample after which a time series is considered stale." - although it begs the question what "stale" means.

Reply:
What about "Time since the last sample after which a time series is considered stale and ignored by expression evaluations."